Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
I'll probably tidy up the code comment a little, do some more testing locally & file that in `brew` as a PR before I roll into bed for the evening.

This is one of the very very few times the repo split is a pain because these changes bump into each other but can't be CI tested simultaneously.